recipes/nolearn/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nolearn" %}
+{% set version = "0.6.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "bab2dc2aeb9560264791298f1d206b78fd9b345cecdd500b34f469d367765158" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Please use `skip: true  # [win or py>36]` similar to https://github.com/conda-forge/lasagne-feedstock/pull/3 because you depend on `lasagne` so you won't be able to build a python 3.7 package if `lasagne` is not available for python 3.7.